### ISSUE TITLE
Billybairn62

### ISSUE THREAD

**Issue by Billybairn62** [nixgates/plugin.video.seren#346]

**Comment by Billybairn62** [comment]
Hi,

Using Seren 2.0.9, on Kodi 18.8

Since the update, I am unable to get new shows. When I search for a show, the title is returned, but no seasons/episodes are shown.

However, Trakt shows in my "Next Up" section all play as normal, with no issues.

Movies seem to be ok.

**Comment by nixgates** [comment]
@Billybairn62 I'm really not sure what you mean here, is there any errors? 

Are you saying you are unable to browse a show?

**Comment by oNeXuSo** [comment]
Hi 
I'm having the same issue in trying to find new TV shows (Seren - Discover TV shows - New TV Shows) since update to 2.0.9, (Kodi version 18.9) 

It gives an error and to check the log. I've attached this part of the log if it can help identify why this is happening.  

Appreciate you are busy with more pressing issues however hope this helps when / if you manage to get some time to investigate.  

[SerenLog.txt](https://github.com/nixgates/plugin.video.seren/files/5773487/SerenLog.txt)

**Comment by nixgates** [comment]
@oNeXuSo please refer to #273 for your issue, I'm not entirely sure this is the same

**Comment by Billybairn62** [comment]
Hi, 

Sorry I wasn't very clear, not very technically inclined! :-)

Since the 2.0.9 update, if I go into Seren, select "Discover TV Shows" then select "Search" and type in the name of a TV show, then "OK", the results show that show (and alternatives). However, If I then select the show I want, it doesn't show any seasons or episodes, I simply get the screen with two dots on it, as if there are no results. I've tried on older shows that I know should have seasons/episodes, but I still get the two dots on selecting the show. I've tried with multiple shows, new and old, but the same thing happens.

This is if I search for a "new" show that is not already in my "Next Up" folder. Those shows are still working, so I'm assuming Trackt and RD are both OK.

There are no error messages that come up during the search process, just the two dots.

Hope this helps!

Thanks!
